annyang.js

```diff
@@ -82,6 +82,12 @@
     return recognition !== undefined;
   };
 
+  // method for logging in developer console when debug mode is on
+  var logMessage = function(text,logStyle){
+  	logStyle = logStyle || debugStyle;
```

Comment:
I would suggest renaming logStyle to something more general like extraParameters, because when they aren't used for styles they contain extra parameters such as matched commands, etc.